tray/actions.go

```diff
@@ -164,3 +165,75 @@ func (ti *Instance) disconnect() bool {
 	}
 	return true
 }
+
+func (ti *Instance) setNotify(flag bool) bool {
+	flagText := "off"
+	if flag {
+		flagText = "on"
+	}
+	resp, err := ti.client.SetNotify(context.Background(), &pb.SetNotifyRequest{
+		Uid:    int64(os.Getuid()),
+		Notify: flag,
+	})
+	if err != nil {
+		log.Printf("%s Setting notifications %s error: %s", internal.ErrorPrefix, flagText, err)
+		ti.notify("Setting notifications %s error: %s", flagText, err)
+		return false
+	}
+
+	switch resp.Type {
+	case internal.CodeConfigError:
+		log.Printf("%s Setting notifications %s error: %s", internal.ErrorPrefix, flagText, "Config file error")
+		ti.notify("Setting notifications %s error: %s", flagText, "Config file error")
+		return false
+	case internal.CodeNothingToDo:
+	case internal.CodeSuccess:
+	}
+
+	_, err = ti.fileshareClient.SetNotifications(context.Background(), &filesharepb.SetNotificationsRequest{Enable: flag})
+	if err != nil {
+		log.Printf("%s Setting fileshare notifications %s error: %s", internal.ErrorPrefix, flagText, err)
```

Comment:
Done.

Reply:
When `ti.client.SetNotify` is successful it means that the setting was changed succesfully in the config and we need to change the displayed status in menu. Fail of `ti.fileshareClient.SetNotifications` means only that fileshared has not been informed about that change, for example because meshnet is disabled and fileshared is not running.